MIGRATION.md

```diff
@@ -6,6 +6,11 @@ API changes in MoveIt releases
 - ServoServer was renamed to ServoNode
 
 ## ROS Noetic
+- `CollisionObject` messages are now defined with a `Pose`, and shapes and subframes are defined relative to the object's pose. This makes it easier to place objects with subframes and multiple shapes in the scene. This causes several changes:
+    - `getFrameTransform()` now returns this pose instead of the first shape's pose.
+    - The Rviz plugin's manipulation tab now uses the object's pose instead of the shape pose to evaluate if object's are in the region of interest.
+    - Planning scene geometry text files (`.scene`) have changed format. Add a line `0 0 0 0 0 0 1` under each line with an asterisk to upgrade old files if required.
+- Static member variable interface of the CollisionDetectorAllocatorTemplate for the string NAME was replaced with a virtual method `getName`.
```

Comment:
do you mind moving this into a section above under rolling because these are things that will not be backported into foxy and are now needed to migrate from foxy to rolling.